app/assets/stylesheets/articles.scss

```diff
@@ -1,3 +1,71 @@
 // Place all the styles related to the Articles controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+
+@import "colours";
+
+h1 {
+  text-align: center;
+}
+
+textarea {
+  width: 90%;
+  height: 150px;
+}
+
+.centre {
+  width: 50%;
+  margin: 0 auto;
+}
+
+.button-container {
+  display: flex;
+  justify-content: center;
+}
+
+.article-menu {
+  display: flex;
+  flex-direction: column;
+}
+
+.article-container {
+  display: flex;
+  justify-content: space-between;
+  background-color: lighten($accent, 50%);
```

Comment:
Given that you use `lighten($accent, 50%);` in several places, this could be its own variable in your `_colours.scss` file.